core_impl/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -299,6 +299,16 @@ void combine(MutableAggregation other, double fraction) {
       checkArgument(this.bucketBoundaries.equals(mutableDistribution.bucketBoundaries),
           "Bucket boundaries should match.");
 
+      // Algorithm for calculating the combination of sum of squared deviations:
+      // https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm.
+      if (this.count + mutableDistribution.count > 0) {
+        double delta = mutableDistribution.mean - this.mean;
+        this.sumOfSquaredDeviations = this.sumOfSquaredDeviations
```

Comment:
Could be a bit more concise (and parallel with the other updates) with +=, but I think it obscures the correspondence to the linked algorithm--your call.

Reply:
I'll prefer to leave this as it is - to be clear and more consistent with the formula.